clients/ts/signalr/src/HubConnectionBuilder.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+import { HttpConnection, IHttpConnectionOptions } from "./HttpConnection";
+import { HubConnection, JsonHubProtocol } from "./HubConnection";
+import { IHubProtocol } from "./IHubProtocol";
+import { ILogger, LogLevel } from "./ILogger";
+import { NullLogger } from "./Loggers";
+import { ConsoleLogger } from "./Utils";
+
+export class HubConnectionBuilder {
+    private protocol: any;
+    private httpConnectionOptions: IHttpConnectionOptions;
+    private url: string;
+    private logger: ILogger;
+
+    public configureLogging(logging: LogLevel | ILogger): HubConnectionBuilder {
```

Comment:
What happens if null is passed in? Needs to be handled or validated to give a nice error msg.